regression/rt_10m_lhdi_grad_closure.c

```diff
@@ -526,6 +529,7 @@ main(int argc, char **argv)
     step += 1;
   }
 
+  gkyl_moment_app_write(app, t_curr, 1000);
```

Comment:
I think this line should be deleted. You're calling write_data immediately after this.